apb.yml

```diff
@@ -214,9 +214,17 @@ _apb_kieserver_startup_strategy: &_apb_kieserver_startup_strategy
   description: "When set to OpenShiftStartupStrategy, allows Process Server to start up independently used shared state from OpenShift API service. The OpenShiftStartupStrategy strategy is still Technology Preview. Default: ControllerBasedStartupStrategy"
   required: true
   type: enum
-  enum: ['ControllerBasedStartupStrategy','OpenShiftStartupStrategy']
+  enum: ['ControllerBasedStartupStrategy', 'OpenShiftStartupStrategy']
   default: ControllerBasedStartupStrategy
   display_group: Process Server
+_apb_kieserver_prometheus_ext_disabled: &_apb_kieserver_prometheus_ext_disabled
+  name: apb_kieserver_prometheus_ext_disabled
+  title: Prometheus Server Extension
+  description: Choose whether to enable or disable the Prometheus Server Extension. (Sets the org.kie.prometheus.server.ext.disabled system property)
+  required: false
+  type: enum
```

Comment:
why not just use a boolean type that will be rendered as a checkbox? You can disable/enable it by default. In this case should be unchecked by default.

Reply:
Because the requirement has 3 states: undefined, true or false. If undefined the env variable shouldn't exist. That's why I opt for adding a dropdown instead a checkbox. But, I have to agree that this makes little sense.

@errantepiphany could you clarify? Don't you think that this requirement is a bit odd?

> The way the templates and code are now, the system property is only set IF a value for the PROMETHEUS_SERVER_EXT_DISABLED parameter is set. We do not default it to anything.

It's a boolean value, so defaulting to FALSE (unchecked), would make sense for this scenario since the prometheus extension would be enabled by default. Doing this in templates would be different from the APB and Operator since they have UI.

Reply:
@ricardozanini I'm okay with the checkbox, and defaulting to unchecked (disabled = false) value.

Reply:
+1